### PR TITLE
✨ [Story video] Google cache validation

### PR DESCRIPTION
**Pull request by mszylkowski**
Added the `cache="google"` attribute to amp-videos inside amp-story, which requires the extension `amp-url-cache`.

### CODE REVIEW

**Review by newmuis — CHANGES_REQUESTED** [review]
Sorry, can we please reach alignment on the comments in https://docs.google.com/document/d/1Ti1JdC8JxXXIKTq6gh5_0N6Odm5WWXgy5vQ-mC7U7dE before proceeding?

**Comment by mszylkowski** [comment]
Yes, I'm reverting the changes unrelated to the validation, and this will stay as draft until we resolve the diffbase PR

File: extensions/amp-video/0.1/test/validator-amp-video-google-cache-story.out [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+|
+|    Unless required by applicable law or agreed to in writing, software
+|    distributed under the License is distributed on an "AS-IS" BASIS,
+|    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+|    See the License for the specific language governing permissions and
+|    limitations under the license.
+|  -->
+|  <!--
+|    Test Description:
+|    Tests for amp-video tag under the google cache.
```
Comment (gmajoulet) [inline extensions/amp-video/0.1/test/validator-amp-video-google-cache-story.out:19]: Plz remove google in these files/filenames

**Review by gmajoulet — APPROVED** [review]

**Review by newmuis — APPROVED** [review]

**Comment by gmajoulet** [comment]
to @ampproject/wg-caching for approval

**Review by MichaelRybak — APPROVED** [review]

File: extensions/amp-cache-url/validator-amp-cache-url.protoascii [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-cache-url
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-cache-url"
+    version: "0.1"
```
Comment (westonruter) [inline extensions/amp-cache-url/validator-amp-cache-url.protoascii:22]: Is there a reason why `latest` was omitted here? See #34636.

**Pull request merged**